programming_examples/channel_examples/worker_to_self/worker_to_self.py

```diff
@@ -47,51 +51,43 @@ def launch_body(a, b):
             @segment(name="seg")
             def segment_body():
 
+                tensor_in_l2 = AllocOp(image_type_l2, [], [])
+                ChannelGet("ChanIn", tensor_in_l2)
```

Comment:
After `tensor_in_l2` is filled with data from getting `ChanIn`, I would expect a put to some L2->L1 channel?

Reply:
I have to do L2->L1 in the herd due to the purpose of this example (see my other comment).

If this not a valid operation?

Reply:
It is valid operation. I didn't understand the intend the design but I get it now.